src/fallbackServiceStub.ts

```diff
@@ -171,6 +175,11 @@ export function generateServiceStub(
             ])
               .then(([ok, buffer]: [boolean, Buffer | ArrayBuffer]) => {
                 const response = responseDecoder(rpc, ok, buffer);
+                if (!callback) {
+                  throw new Error(
+                    'Internal error: callback is not provided for non-streaming call.'
```

Comment:
Should we have a test case that looks specifically for this error message?

Reply:
I actually decided to remove that error message since there is no way for the user to handle or catch it.